src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/TimeDependentTripleGaussian.hpp

```diff
@@ -85,14 +90,30 @@ class TimeDependentTripleGaussian : public DampingFunction<3, Frame::Grid> {
   template <typename Group>
   struct Center {
     using group = Group;
-    using type = std::array<double, 3>;
+    using type = tmpl::conditional_t<
+        std::is_same_v<Group, Gaussian<3>>, std::array<double, 3>,
+        Options::Auto<std::array<double, 3>, Options::AutoLabel::None>>;
```

Comment:
Why `None` here instead of `Auto`? Not really sure what a "none" center is for a Gaussian

Reply:
Agreed, changed.